completion/available/npm.completion.bash

```diff
@@ -1,8 +1,9 @@
 #!/usr/bin/env bash
 
 # npm (Node Package Manager) completion
+# https://docs.npmjs.com/cli/completion
 
 if command -v npm &>/dev/null
 then
-  eval "$(npm completion)"
+  source <(npm completion)
```

Comment:
Not really sure, but probably the problem is the Bash version, @nwinkler is using the  v3.X. So reverting to the old form is the best option to don't break backward compatibility.